### PR TITLE
Gov module restore

### PR DESCRIPTION
**Pull request by bragaz**
## Description
This PR restore `gov` module and its tests + fix the `DefaultGasValue` of sim tests
that was wrong
## Checklist
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Wrote unit tests.
- [x] Wrote integration tests (simulation & CLI). 
- [ ] Updated the documentation. 
- [x] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Version 0.7.0
 - Implemented benchmarks tests (#126)
+- Restore `gov` module (#173)
```
Comment (RiccardoM) [inline CHANGELOG.md:3]:
```suggestion
- Re-introduced the on-chain government module (#173)
```

File: app/app.go [hunk 2]
```diff
@@ -186,6 +193,15 @@ func NewDesmosApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 		app.subspaces[crisis.ModuleName], invCheckPeriod, app.SupplyKeeper, auth.FeeCollectorName,
 	)
 
+	// register the proposal types
+	govRouter := gov.NewRouter()
+	govRouter.AddRoute(gov.RouterKey, gov.ProposalHandler).
+		AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(app.paramsKeeper)).
+		AddRoute(distr.RouterKey, distr.NewCommunityPoolSpendProposalHandler(app.DistrKeeper))
```
Comment (RiccardoM) [inline app/app.go:200]: I think we can avoid implementing the `NewCommunityPoolSpendProposal` for now. 

```suggestion
	govRouter.AddRoute(gov.RouterKey, gov.ProposalHandler).
		AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(app.paramsKeeper))
```

File: x/posts/internal/simulation/operations.go [hunk 3]
```diff
@@ -19,7 +19,7 @@ const (
 	OpWeightMsgAnswerPoll       = "op_weight_msg_answer_poll"
 	OpWeightMsgRegisterReaction = "op_weight_msg_register_reaction"
 
-	DefaultGasValue = 5000000
+	DefaultGasValue = 2000000
```
Comment (RiccardoM) [inline x/posts/internal/simulation/operations.go:22]: Can we try changing this to `200000` instead? It should work

```suggestion
	DefaultGasValue = 200000
```

File: x/profile/internal/simulation/operations.go [hunk 4]
```diff
@@ -15,7 +15,7 @@ const (
 	OpWeightMsgSaveProfile   = "op_weight_msg_save_profile"
 	OpWeightMsgDeleteProfile = "op_weight_msg_delete_profile"
 
-	DefaultGasValue = 200000
+	DefaultGasValue = 2000000
```
Comment (RiccardoM) [inline x/profile/internal/simulation/operations.go:18]: We should probably make this `200000` instead as it should work properly and this is the default value that clients will use as well

```suggestion
	DefaultGasValue = 200000
```

**Comment by bragaz** [comment]
I'm closing this since I've open the other PR because of sim-tests not passing on the CI but passing on my laptop locally.

**Pull request closed without merging**